packages/block-editor/src/components/simulate-media-query/index.js

```diff
@@ -74,6 +75,13 @@ export default function useSimulatedMediaQuery( marker, width ) {
 				++ruleIndex
 			) {
 				const rule = styleSheet.cssRules[ ruleIndex ];
+				if (
+					rule.type !== window.CSSRule.STYLE_RULE &&
+					rule.type !== window.CSSRule.MEDIA_RULE
```

Comment:
It is slightly sketchy to not be guarding `window` here, but if we assume that `getStyleSheetsThatMatchHostname` is returning an empty array for any non-DOM environments, then this logic (within `styleSheets.forEach`) would never be reached.